lib/jazzy/config.rb

```diff
@@ -400,6 +401,12 @@ def print_attr_description(attr)
       end
     end
 
+    class PathnameArray < Array
```

Comment:
I don’t think that’s an improvement. `PathnameArray` subclass removed, `is_path?` option added:

    def expand_paths(base_path)
      self.class.all_config_attrs.each do |attr|
        val = attr.get(self)
        if attr. is_path?
            new_val = case val
              when Array:
                val.expand_path(base_path)
              when Pathname:
                val.map { |path| path.expand_path(base_path) }
            end
            attr.set_raw(self, new_val) if new_val
          end
        end
      end
    end

That type-based case is a much stronger smell for my money.